Assets/_BuildingBlocks/LayerStyles/Scripts/StylingRule.cs

```diff
@@ -0,0 +1,31 @@
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+
+namespace Netherlands3D.LayerStyles
+{
+    [DataContract(Namespace = "https://netherlands3d.eu/schemas/projects/layers/styling", Name = "StylingRule")]
+    public class StylingRule
+    {
+        [DataMember(Name = "name")] public string Name { get; private set; }
+        [DataMember(Name = "symbolizer")] public Symbolizer Symbolizer { get; } = new();
+
+        [DataMember(Name = "selector")] public Expression Selector { get; private set; }
+
+        [JsonConstructor]
+        private StylingRule()
```

Comment:
de JSONConstructor moet niks doen?

Reply:
Leuk dat je dit vraagt: bij een StylingRule wil ik dat verplicht een naam wordt opgegeven als je in code de constructor gebruikt; dus ik heb bewust geen default constructor om dat te kunnen afdwingen. Maar als ik niet een default constructor opneem met JsonConstructor, dan gaat het bij het deserializen niet helemaal goed. Door een private default constructor op te nemen, en die als JsonConstructor te markeren, doet Newtonsoft het object inladen en alle properties assignen zoals bedoeld